docs/tutorials/embedding/MASE.ipynb

```diff
@@ -0,0 +1,377 @@
+    "0.1 & 0.3\n",
+    "\\end{bmatrix}\n",
+    "\\end{align*}\n",
+    "\n",
+    "The only difference between the two is the within-block probability for the second block. We sample $G_1 \\sim \\text{SBM}(n, P_1)$ and $G_2 \\sim \\text{SBM}(n, P_2)$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graspologic.simulations import sbm\n",
+    "\n",
+    "n = [25, 25]\n",
+    "P1 = [[0.3, 0.1],\n",
+    "      [0.1, 0.7]]\n",
+    "P2 = [[0.3, 0.1],\n",
+    "      [0.1, 0.3]]\n",
+    "\n",
+    "np.random.seed(8)\n",
+    "G1 = sbm(n, P1)\n",
+    "G2 = sbm(n, P2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualize the graphs using heatmap\n",
+    "\n",
+    "We visualize the sampled graphs using the ``heatmap()`` function. ``heatmap()`` will plot the adjacency matrix, where the colors represent the weight of each edge. In this case, we have binary graphs so the values will be either 0 or 1. \n",
+    "\n",
+    "We see that there is clear block structure to the graphs. Furthermore, the lower right quarter of $G_1$, representing the within-group connections for the second group, is more dense than that of $G_2$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graspologic.plot import heatmap\n",
+    "\n",
+    "g1 = heatmap(G1, figsize=(7,7), title=\"Visualization of Graph 1\")\n",
+    "g2 = heatmap(G2, figsize=(7,7), title=\"Visualization of Graph 2\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Embed the two graphs using MASE"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Just as ASE fits a single input graph to the Random Dot Product Graph (RDPG) model, MASE fits a sample of graphs to the common subspace independent-edge (COSIE) model, which is an extension of RDPG for multiple graphs. The COSIE model assumes that all of the graphs have important shared properties, so all graphs are derived from a common subspace with orthonormal basis $V$.  But, COSIE also allows the connection probabilities between each vertex to vary from one graph to the next, so each graph $A^{(i)}$ has a corresponding score matrix $R^{(i)}$.  As a result, COSIE facilitates comparison due to the common subspace, while retaining sufficient flexibility to represent heterogeneous collections of graphs, due to the different score matrices."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since we know that 2 clusters should be embedded into 2 dimensions, we specify the number of components as 2. Then, the ``fit_transform()`` method returns the estimated invariant subspace $\\hat{V}$, which we can see has 50 entries corresponding to the 50 vertices embedded in 2 dimensions."
```

Comment:
"Since we know that 2 clusters should be embedded into 2 dimensions" technically, this isn't correct (because of 
```
[[p^2, pq],
[pq, q^2]]
```
 model), but i'm not sure if being explicit is better here..

Reply:
Could you explain?

Reply:
Ok, that makes sense, but it's a special case...
Would it suffice to say, "Since we know that, in general, 2 clusters can be faithfully represented in 2 dimensions, we specify...."?